source/_components/media_player.enigma2.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Volumio Media Player"
+description: "How to set up the Enigma2 media player platform"
+date: 2017-04-07
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Media Player
+ha_release: 0.42
+---
+
+The `Enigma2` platform allows you to control [Enigma2](http://wiki.blue-panel.com/index.php/Enigma2_(en)) linux based settopboxes through web API
+from Home Assistant.
```

Comment:
There is no need to add line breaks. The renderer will take care about this for you.